cbo: Update base images for the e2e-metal-assisted-flow

lranjbar opened the pull request:
Updating the base images for assisted installer since they have been moved to the ocp-kni namespace.

Image promotion settings:
https://github.com/openshift/release/blob/master/ci-operator/config/openshift/assisted-service/openshift-assisted-service-master.yaml#L43-L45
https://github.com/openshift/release/blob/master/ci-operator/config/openshift/assisted-installer/openshift-assisted-installer-master.yaml#L52-L54
https://github.com/openshift/release/blob/master/ci-operator/config/openshift/assisted-installer-agent/openshift-assisted-installer-agent-master.yaml#L42-L44

asalkeld reviewed (commented):
seems ok, but broken tests

asalkeld commented:
/lgtm cancel

asalkeld commented:
/retest

YuviGold commented:
/retest
/lgtm

andfasano commented:
Is this a new requirement for assisted jobs? No description or reference have been provided

lranjbar commented:
> Is this a new requirement for assisted jobs? No description or reference have been provided

I updated the description for the reasoning. I noticed that the failing e2e-metal-assisted job was using out of date images. This is because without this it looks for them in the ocp namespace. Those images in the ocp namespace are out of date since we changed the namespace of the image promotion. Also this flow uses all the components of assisted so you need all the images to run it successfuly.

andfasano commented:
> > Is this a new requirement for assisted jobs? No description or reference have been provided
> 
> I updated the description for the reasoning. I noticed that the failing e2e-metal-assisted job was using out of date images. This is because without this it looks for them in the ocp namespace. Those images in the ocp namespace are out of date since we changed the namespace of the image promotion. Also this flow uses all the components of assisted so you need all the images to run it successfuly.

Thanks for clarifying! I guess you're referring to the `e2e-metal-assisted` test, which unfortunately seems not to be triggered by the rehearsal

YuviGold commented:
> > > Is this a new requirement for assisted jobs? No description or reference have been provided
> > 
> > 
> > I updated the description for the reasoning. I noticed that the failing e2e-metal-assisted job was using out of date images. This is because without this it looks for them in the ocp namespace. Those images in the ocp namespace are out of date since we changed the namespace of the image promotion. Also this flow uses all the components of assisted so you need all the images to run it successfuly.
> 
> Thanks for clarifying! I guess you're referring to the `e2e-metal-assisted` test, which unfortunately seems not to be triggered by the rehearsal

The `e2e-metal-assisted` job has passed now :) It isn't triggered by default on purpose, it is declared as `optional: true` and `always_runs: false`

andfasano commented:
/approve
